src/forms/reset-password-popup/messages.js

```diff
@@ -81,7 +81,7 @@ const messages = defineMessages({
   },
   passwordValidationMessage: {
     id: 'password.validation.message',
-    defaultMessage: 'Password needs to include:',
+    defaultMessage: 'Password criteria has not been met',
```

Comment:
Can we remove this message object and use only [this](https://github.com/edx/frontend-component-authn-edx/pull/94/files#diff-c32790c83f801f924ad4984dc0be6b63fe26c6451483063f132ba6a4c25e8dffL29) message object to remove the redundant code?

Reply:
We need to keep each message separate, just like the others. That is why we have done it this way.